app/src/main/java/lu/geoportail/map/MainActivity.kt

```diff
@@ -15,7 +15,9 @@ import android.webkit.WebChromeClient
 
 class MainActivity : Activity() {
     // val websiteUrl = "https://map.geoportail.lu/?localforage=android" // production
-    private val websiteUrl = "https://map.geoportail.lu?localforage=android&applogin=yes&embeddedserver=127.0.0.1:8766/static&embeddedserverprotocol=http&version=3"
+    // private val websiteUrl = "https://map.geoportail.lu?localforage=android&applogin=yes&embeddedserver=127.0.0.1:8766/static&embeddedserverprotocol=http&version=3"
+    private val websiteUrl = "http://10.0.2.2:8080/dev/main.html"
+    //private val websiteUrl = "https://migration.geoportail.lu?localforage=android&applogin=yes&embeddedserver=127.0.0.1:8766/static&embeddedserverprotocol=http&version=3"
```

Comment:
Maybe a conditional that will check if we are in debug mode or release mode and update the website url accordingly (like in iOS) : https://github.com/Geoportail-Luxembourg/luxmob-ios/blob/upgrade-project/geoportail.lu/ViewController.swift#L20-L26